.storybook/StoryThemeProvider.tsx

```diff
@@ -0,0 +1,9 @@
+import { ThemeProvider } from '@terra-ui-packages/components';
+import { PropsWithChildren } from 'react';
+import { defaultBrand } from 'src/libs/brands';
+
+const StoryThemeProvider = (props: PropsWithChildren) => {
+  const { children } = props;
+  return ThemeProvider({ theme: defaultBrand.theme, children });
```

Comment:
Should this be rendering ThemeProvider?

```suggestion
  return <ThemeProvider theme={defaultBrand.theme} children={children} />;
```

Reply:
Sure, that makes sense… though it doesn't seem to matter either way.

Reply:
I think the function call only worked because ThemeProvider happens to not use hooks.

Reply:
FYI, eslint doesn't like passing children that way. "Do not pass children as props. Instead, nest children between the opening and closing tags."

Reply:
👍 Yeah, that's more conventional.